src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.FixAllProvider.cs

```diff
@@ -26,7 +26,7 @@ public async override Task<CodeAction> GetFixAsync(FixAllContext fixAllContext)
                     return null;
                 }
 
-                var batchFixer = WellKnownFixAllProviders.BatchFixer;
+                var batchFixer = BatchFixAllProvider.Instance;
```

Comment:
This change was needed as the `GetFixAsync` API below was moved from being an internal virtual API on public FixAllProvider type to an internal API directly on `BatchFixAllProvider` as this was the only place that API was being invoked.

Reply:
does thsi change behavior at all?  or is it the exact same impl?